Timeline.Core/Timeline.cs

```diff
@@ -3877,9 +3899,9 @@ private void OnDuplicate(ObjectCtrlInfo source, ObjectCtrlInfo destination)
                 }
             }, 20);
         }
-        #endregion
+#endregion
 
-        #region Patches
```

Comment:
Removing these spaces(line 3880,3902,4074) was unnecessary.
I wonder if it would be better to put a PR to revert this.

Reply:
It's fine if it does look bad it'll get taken care of next time someone works on this part of the code.